build-logic/build.gradle.kts

```diff
@@ -98,6 +98,10 @@ gradlePlugin {
             id = "droidkaigi.primitive.detekt"
             implementationClass = "io.github.droidkaigi.confsched2023.primitive.DetektPlugin"
         }
+        register("oss-licenses") {
+            id = "droidkaigi.primitive.osslicenses"
```

Comment:
Thanks! We might have to use `droidkaigi.primitive.android.osslicenses` because the license list plugin is based on AGP plugin.